opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -895,10 +904,7 @@ namespace Opm {
         } else if (setCount != 0) {
             std::string msg = "BOX modifiers on keywords must be either "
                 "specified completely or not at all. Ignoring.";
-            OpmLog::addMessage(Log::MessageType::Error,
-                               Log::fileMessage(deckKeyword.getFileName(),
-                                                deckKeyword.getLineNumber(),
-                                                msg));
+            m_intGridProperties.getMessageContainer().error(deckKeyword.getFileName() + std::to_string(deckKeyword.getLineNumber()) + msg);
```

Comment:
But [a different MessageContainer.error](https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/Parser/MessageContainer.hpp#L70) function has the following signature
> `void error(const std::string& msg, const std::string& filename, const size_t lineno);`